pallet/crowdsale/src/lib.rs

```diff
@@ -203,14 +203,14 @@ pub mod pallet {
 		InvalidCrowdsaleStatus,
 		/// Crowdsale is not enabled
 		CrowdsaleNotEnabled,
+		/// The soft cap price must be greater than zero
+		InvalidSoftCap,
```

Comment:
nit: ambigious if softcap total or softcap price
```suggestion
		InvalidSoftCapPrice,
```

Reply:
https://github.com/futureversecom/trn-seed/pull/799/commits/3d396a7ff08f15ec8217f78889223bb0ef24e21c